### ISSUE TITLE
Static content URLs not right in Heroku deployment

### ISSUE THREAD

**Issue by guillegette** [FoundersAS/walkiebot#4]
Hi team,

First of all, what an awesome tool. Sad to see you guys take this down but totally understandable. I went ahead and run the deployment in Heroku and everything went well but once you visit the app URL I can see that 2 JS files don't have the right path.

![image](https://user-images.githubusercontent.com/938027/40077379-5730ac3a-5836-11e8-8131-5a4f7e05c45b.png)

Any ideas?
Deployment URL: https://workast-walkiebot.herokuapp.com/

**Comment by alexagranov** [comment]
So I noticed this as well, even just trying `yarn start` locally to serve the `server/index.js` file.

Then I took another look at the Quick Start and that is referring to `yarn dev` which actually runs `bin/start-dev.sh` to start a few processes locally - one of them is actually booting webpack-dev server via `devserver.js` (on 8005) while nodemon is used to kick up another server on 8000 to service the endpoints (proxied to from `devserver.js`).  I opened a ticket/PR to get some clarification if possible in #2. 

Basically, I don't think `server/index.js` is ever used to serve static files, only `dev-index.ejs` is used for that (I think).

**Comment by dinoshauer** [comment]
Hiya! Just fixed this in https://github.com/FoundersAS/walkiebot/commit/9aa5d6c1dd066f94281cf460b2918a9722f28a69 - You may have to do a redeploy on heroku or recreate the app, not sure how that works with heroku. In any case it's probably worth forking this repo after creating the heroku app and setting it up to track your own fork so you can deploy it directly from git :)

@alexagranov the `catchAll` and `dist` handlers are used to serve some of that content, specifically https://github.com/FoundersAS/walkiebot/blob/master/server/handlers/resources.js is used to serve all static files :)
